sdk/search/azure-search-documents/src/test/README.md

```diff
@@ -72,20 +72,3 @@ To run tests in Playback mode:
 
 1. Set the value of the **AZURE_TEST_MODE** environment variable to *'PLAYBACK'* (or delete it). You can also delete all other environment variables.
 2. Run the tests.
```

Comment:
Is this intentional? To remove the entire section?

Reply:
which section?

Reply:
Oh ok now I see. You are referring to the examples.  Yes this one is intentional, discussed with @alzimmermsft  as this looks like from some old readme which is no longer applicable

Reply:
@conniey @alzimmermsft If you want to update the content here, please feel free to do so in a separate PR